testing/str_test.py

```diff
@@ -12,7 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# pylint: disable=redefined-outer-name
+# pylint: disable=redefined-outer-name,W0703
```

Comment:
It was for this clause, but may be I should do that way in first place. Will remove.
`except Exception as e`